src/DataAccess/Sofort/Transfer/Client.php

```diff
@@ -15,14 +15,15 @@ class Client {
 	/**
 	 * @var Sofortueberweisung
 	 */
-	private $api;
+	private Sofortueberweisung $api;
```

Comment:
Please remove the docblock when adding prop types.

BTW, we should make that a Sniff

Reply:
Whoops, didn't even see the comment :joy: